Readme.md

```diff
@@ -64,6 +64,19 @@ will inform you if a bundled library is used or if you need to install any
 missing packages yourself.
 
 ### macOS Build Steps:
+A script is provided to build Universal binaries supporting both x64 and ARM in the same
+application bundle using the following steps:
+
+1. `mkdir build`
+2. `cd build`
+3. `python ../BuildMacOSUniveralBinary.py`
```

Comment:
There's an unfortunate typo here!
```suggestion
3. `python ../BuildMacOSUniversalBinary.py`
```

Reply:
Thanks for pointing that out! I fixed it in the latest commit.